### DISCUSSION TITLE
Combustion process - natural gas

### DISCUSSION THREAD

**Discussion by Ferdisss** [firemodels/fds#12618] [Q&A]
Hi, 
I want to make a simple simulation of a fire where I look at the heat flux from the fire. I want to use natural gas as my source, and I have written the following code from an example in the FDS user guide: 

----Defining natural gas:----

&SPEC ID='METHANE', LUMPED_COMPONENT_ONLY=T /
&SPEC ID='NITROGEN', LUMPED_COMPONENT_ONLY=T /
&SPEC ID='ETHYLENE', LUMPED_COMPONENT_ONLY=T /
&SPEC ID='CARBON DIOXIDE',LUMPED_COMPONENT_ONLY=T /
&SPEC ID='Natural gas'
	SPEC_ID(1)='METHANE', VOLUME_FRACTION(1)=92.2
	SPEC_ID(3)='NITROGEN', VOLUME_FRACTION(3)= 3.9
	SPEC_ID(2)='ETHYLENE', VOLUME_FRACTION(2)= 3.3
	SPEC_ID(4)='CARBON DIOXIDE',VOLUME_FRACTION(4)= 0.6 /

---- Defining natural gas as reaction fuel ----

&REAC FUEL='natural gas'
SOOT_YIELD=0.01 /

My question is - **do you know if the heat flux from this fire is dependent on whether the fuel is liquid or gas? And if so, do you know if the program treats it as liquid or gas? I have defined my initial temperature for the program as 25 degrees celsius: (&MISC TMPA = 25 /) - but is the program 'smart enough' to change the form of the fuel from this?** 

Thank you! 
Kind regards

**Comment by rmcdermo** [comment]
All those components are in the gas phase at 25 C and 1 atm.  There is a discussion in the user guide about liquids.  See "Liquid Fuels" and "Defining Liquid Properties".

The REAC line in FDS describes a gas phase reaction only.  Any liquid fuels (e.g., kerosene) have to first evaporate before mixing with air to burn.  Liquids in FDS are represented by a SURF attached to a boundary, an OBST, or a particle.

However, in this situation, there is no need to worry about liquids.  Yes, FDS knows those components are gases; they are in a table of predefined species, see Appendix A of the user guide.

**Reply by Ferdisss** [reply]
Thank you!
